data-otservbr-global/migrations/28.lua

```diff
@@ -1,3 +1,5 @@
 function onUpdateDatabase()
-	return false -- true = There are others migrations file | false = this is the last migration file
+	Spdlog.info("Updating database to version 29 (transfer coins)")
+	db.query("ALTER TABLE `accounts` ADD `transfercoins` int NOT NULL DEFAULT '0';")
```

Comment:
I think we can reuse the tournament fields, which currently have no use

Reply:
yes, I will do changes on it.